### PR TITLE
Desktop branch pull request

### PR DESCRIPTION
**Pull request by codehass**
In this pull request : 
-  Add HTML code for the desktop version.
-  Add CSS style for the desktop version.

Thank you for reviewing my pull request.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -147,96 +173,112 @@ <h2>About</h2>
             aliquip ea eiusmod
           </p>
           <div class="about-butt">
-            <p>Get my CV</p>
+            <p class="about-butt-text">Get my CV</p>
             <img src="imgs/ic_download.png" alt="" />
           </div>
```
Comment (TuyishimireEric) [inline index.html:188]: - [ ] I would recommend replacing this ```p``` tag with a ```button``` or an ```anchor``` as the ```<a>``` tag defines a hyperlink, which is used to link from one page to another and also can be used to download embedded files on your webpage, 👍 👍

File: index.html [hunk 2]
```diff
@@ -54,13 +68,13 @@ <h2>Projects</h2>
               <li>JavaScript</li>
             </ul>
             <div class="project-butt">
-              <p>See this project</p>
+              <p class="text-butt">See this project</p>
```
Comment (TuyishimireEric) [inline index.html]: - [ ] **Good job done so far** 👏. Please implement the hover effect styles for all your buttons. Remember it is important that you try to replicate the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=67%3A553) as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

you can Kindly follow the given button interaction styling on all your buttons :joy:
image

  - Please I would recommend replacing this ```p``` tag with a ```button``` or an ```anchor``` as the ```<a>``` tag defines a hyperlink, which is used to link from one page to another and also can be used to download embedded files on your webpage, 👍 👍 
  - Kindly try to add the allow icon on this button as given on Figma design.
   
**Your design** | **Figma design**
| --- | --- |
image | image

**Review by TuyishimireEric — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :recycle: 
### Hi  @codehass     👋 👋 

#### I'm @TuyishimireEric, Your code reviewer,
Good job so far! 🥇 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights 
:heavy_check_mark: No Linter Errors :+1: 
:heavy_check_mark: Professional PR :+1: 
:heavy_check_mark: Good design :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
**NA**
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mwanawabangona — APPROVED** [review]
Hi @codehass 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implementing the requested changes!! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**